src/MainPage/MainPage.js

```diff
@@ -1,24 +1,39 @@
-import React, { useEffect } from "react";
-import { connect } from "react-redux";
-import PropTypes from "prop-types";
-import { Link } from "react-router-dom";
-import * as actions from "../redux/actions/actions";
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import * as actions from '../redux/actions/actions';
 
 const MainPage = ({ postsData, dispatch }) => {
   useEffect(() => {
-    if (!postsData) dispatch(actions.GetPosts());
+    if (!postsData) dispatch(actions.getUsersAndPosts());
   });
   return (
     <>
-    <h2 className="text-center"><strong>Last 5 posts</strong></h2>
+      <h2 className="text-center">
+        <strong>Last 5 posts</strong>
+      </h2>
       {postsData
-        ? postsData.slice(0, 4).map((item, index) => (
+        ? postsData.slice(0, 5).map((item, index) => (

```

Comment:
There should be a function in store (kind of getter in vue) that returns last 5 items from the available data.
I mean you shouldn't perform it here